unlock-app/src/components/interface/keychain/KeyMetadataModal.tsx

```diff
@@ -92,31 +133,52 @@ export const KeyMetadataModal: React.FC<MetadataModalProps> = ({
             {`${lock.name} - Metadata`}{' '}
           </span>
           {hasValues ? (
-            <form action="">
+            <form
+              onSubmit={handleSubmit(onUpdateMetadata)}
+              className="flex flex-col gap-3"
+            >
               {values?.map(([key, value], index) => {
                 return (
-                  <Input
-                    label={key}
-                    disabled={true}
-                    key={index}
-                    {...register(key, {
-                      value,
-                    })}
-                  />
+                  <div key={index}>
+                    <Input
+                      label={key}
+                      key={index}
+                      disabled={updating}
+                      {...register(key, {
+                        value,
+                        required: true,
```

Comment:
Set all fields as required, to avoid users emptying some fields.
There is a way to get the required fields? 

actually is only set in checkout flow with `metadataInputs`? 
@julien51  @searchableguy